core/primitives/block.hpp

```diff
@@ -21,4 +21,11 @@ namespace kagome::primitives {
 
 }  // namespace kagome::primitives
 
+namespace kagome::scale {
+  template <class Stream>
+  Stream &operator<<(Stream &s, const primitives::Block &b) {
```

Comment:
this operator should be in the same namespace as Block

Reply:
fixed